### PR TITLE
Add yamllint config to template

### LINKED ISSUE #1407: Add .yamllint.yaml to avoid error when lines are longer than 80 characters
### Description of feature

I think that after bumping the new node version and the yamllint update in #1398 , we will need to include a `.yamllint.yaml`similarly to what we do with `.markdownlint.yml` here: https://github.com/nf-core/tools/blob/e563811adb292b07ec933a52611e937ce9ef5481/nf_core/pipeline-template/.markdownlint.yml#L3 and it is explained in yamllnt [documentation](https://yamllint.readthedocs.io/en/stable/configuration.html#configuration)

Otherwise, I tried to use the new `linting.yml` on [this](https://github.com/JoseEspinosa/nf-core-atacseq/runs/5144812371?check_suite_focus=true) draft PR and I am getting plenty of errors for lines longer than 80 characters for many of the yaml configuration files on .github/workflows and also others.

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Adds a `.yamllint.yml` config to avoid `line too long` and `missing document start "---"` errors.

Closes #1407 

## PR checklist

 - [x] This comment contains a description of changes (with reason)
 - [x] `CHANGELOG.md` is updated

### CODE REVIEW

**Comment by mashehu** [comment]
For completeness sake, should we also have yamllint for the tools repo itself?

**Comment by maxulysse** [comment]
> For completeness sake, should we also have yamllint for the tools repo itself?

I'd say even for all repos with yml files

**Comment by mashehu** [comment]
Which ones are left? Modules has it. nfco.re not yet.

**Comment by maxulysse** [comment]
> Which ones are left? Modules has it. nfco.re not yet.

technically every repo with some GHA has some yml, so configs could need that as well

**Comment by maxulysse** [comment]
While we're at it, we should probably add it to our extension pack on vs code: https://github.com/nf-core/vscode-extensionpack

File: nf_core/pipeline-template/.github/workflows/linting.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Install yaml-lint
         run: npm install -g yaml-lint
       - name: Run yaml-lint
-        run: yamllint $(find ${GITHUB_WORKSPACE} -type f -name "*.yml" -o -name "*.yaml")
+        run: yamllint $(find ${GITHUB_WORKSPACE} -type f -name "*.yml" -o -name "*.yaml") -c ${GITHUB_WORKSPACE}/.markdownlint.yml
```
[the remark this answers is not in the document]
Comment (JoseEspinosa) [reply nf_core/pipeline-template/.github/workflows/linting.yml:68]: Ups :eyes:

**Comment by JoseEspinosa** [comment]
Will add it to tools in this same PR

**Review by maxulysse — APPROVED** [review]

**Pull request merged**